UI/window-basic-main-outputs.cpp

```diff
@@ -1193,6 +1193,8 @@ bool SimpleOutput::StartStreaming(obs_service_t *service)
 		config_get_bool(main->Config(), "Output", "DelayPreserve");
 	const char *bindIP =
 		config_get_string(main->Config(), "Output", "BindIP");
+	const char *ipVersion =
+		config_get_string(main->Config(), "Output", "ipVersion");
```

Comment:
```suggestion
		config_get_string(main->Config(), "Output", "IPVersion");
```
Should be the same everywhere